### DISCUSSION TITLE
How to use master version?

### DISCUSSION THREAD

**Discussion by MartinRajniak** [GitLiveApp/firebase-kotlin-sdk#227] [Q&A] [answered]
Hi,
is there any way I can use master version?

So that I don't have to wait for release and try master whenever there is something new?

I assume building locally and using that would work.
I know how to work with jvm binaries - I am just not sure how to go about iOS ones.

Any tips?

**Comment by thejoeba** [comment, marked as answer]
One way to accomplish this is to download the repo, and use gradle plugin [maven-publish](https://docs.gradle.org/current/userguide/publishing_maven.html). This will save the library to your mavenLocal() repository (~/.m2/).
